config/locales/en/providers.yml

```diff
@@ -25,6 +25,19 @@ en:
         secure_link_para: Send an email to your client with a secure link to the online assessment.
         send_client_link: Send link
         title: Send your client a link to access the service
+    address_lookups:
```

Comment:
You haven't added an address_lookup page so the whole section being in the PR seems a bit odd 🤔 
It could be a rebase/legacy import?